contracts/sol6/Dao/mock/MockKyberDaoMoreGetters.sol

```diff
@@ -33,7 +31,7 @@ contract MockKyberDaoMoreGetters is KyberDAO {
     }
 
     function replaceStakingContract(address _staking) public {
```

Comment:
function name not clear.
maybe fetchStakingCotractAddress

Reply:
oh it is for changing staking contract to be a normal address, so we can call some functions in DAO that require sender is staking.

Reply:
Oh
OK